slang-checks/src/test/resources/com/sonarsource/slang/checks/UnusedFunctionParameter.slang

```diff
@@ -0,0 +1,22 @@
+void fun fooBar() {}  // OK
+
+void fun fooBar(int a) {}  // Noncompliant {{Remove this unused function parameter "a".}}
+//              ^^^^^
+
+void fun fooBar(int a, int b) {}  // Noncompliant {{Remove these unused function parameters.}}
+//              ^^^^^  ^^^^^<
+
+void fun fooBar(int a, int b) {   // Noncompliant {{Remove this unused function parameter "b".}}
+//                     ^^^^^
+  x = a;
+}
+
+void fun fooBar(int a, int b) {   // Noncompliant {{Remove this unused function parameter "a".}}
```

Comment:
We could have one of these test with a parameter without type to ensure it works in that case too